tests/e2e/config_secret_utils.go

```diff
@@ -654,3 +665,29 @@ func userRoleMap() map[string]string {
 
 	return roleMap
 }
+
+// changeTestUserPassword util method is use for changing testuser vcenter login password
+func changeTestUserPassword(masterIp string, testUser string, testUserPassword string) error {
+	changeUserPassword := govcLoginCmd() + "govc sso.user.update -p " + testUserPassword + " " + testUser
+	framework.Logf("Password change for testuser: %s ", changeUserPassword)
+	result, err := sshExec(sshClientConfig, masterIp, changeUserPassword)
+	if err != nil && result.Code != 0 {
+		fssh.LogResult(result)
+		return fmt.Errorf("couldn't execute command: %s on host: %v , error: %s",
+			changeUserPassword, masterIp, err)
+	}
+	return nil
+}
+
+// getVcenterHostName util method is use to fetch the vcenter hostname
+func getVcenterHostName(masterIp string, vcenterIp string) (string, error) {
```

Comment:
why do you need to run it on k8s master, you should be able to do it from worker/mac as well, just use `exec.Command()`

Reply:
Ack. Fixed